src/v/redpanda/admin/api-doc/broker.json

```diff
@@ -440,6 +462,13 @@
                         "type": "partition_reconfiguration_status"
                     },
                     "description": "Array of partition reconfiguration statues"
+                },
+                "reallocation_failure_details": {
+                    "type": "array",
+                    "items": {
+                        "type": "reallocation_failure_details"
+                    },
+                    "description": "Array containing allocation failure details, this is helpful to diagnose node decommissioning problems."
                 }
```

Comment:
I think this requires an rpk change that reports "ALLOCATION FAILURES" section: cc: @daisukebe